src/junitParser.ts

```diff
@@ -123,24 +121,26 @@ async function parseTestReports(checkName: string, summary: string, reportPathsG
     const globber = await glob.create(reportPathsGlob);
     for await (const file of globber.globGenerator()) {
         core.info(`Parsing report file: ${file}`);
-
-        const { totalCount, skipped, annotations, failedEvaluating, name, fileName } = await parseFile(file, projectTokenDictionaryStrs);
-        if (totalCount === 0) {
-            continue;
+        const suites = await parseFile(file, projectTokenDictionaryStrs);
+        for (const suite of suites) {
```

Comment:
maybe this could later be re-written as an async generator, but for now - this works for me